packages/storybook/package.json

```diff
@@ -19,12 +19,14 @@
     "@storybook/addon-docs": "^6.4.19",
     "@storybook/addon-essentials": "^6.4.19",
     "@storybook/addon-links": "^6.4.19",
+    "@storybook/builder-webpack5": "^6.4.19",
+    "@storybook/manager-webpack5": "^6.4.19",
     "@storybook/react": "^6.4.19",
     "@whitespace/storybook-addon-html": "^5.0.0",
     "babel-loader": "^8.2.2",
     "css-loader": "^5.2.4",
-    "node-sass": "^6.0.0",
+    "node-sass": "^7.0.0",
     "react-dom": "^17.0.2",
-    "sass-loader": "^10.2.1"
+    "sass-loader": "^12.4.0"
```

Comment:
I don't quite remember all of the details around the sass dependencies. Did we do some earlier work because we couldn't get to `sass-loader` version 12 without other issues?

Reply:
@bkjohnson yes so in order for us to get node-sass to v7.0.0, which is really what started this all because of the security issue, we need to use sass-loader v12.4.0 and to use sass-loader v12.4.0 we have to use webpack 5 in storybook